internal/xds/translator/cluster.go

```diff
@@ -81,9 +84,13 @@ func buildEndpointType(settings []*ir.DestinationSetting) EndpointType {
 }
 
 func buildXdsCluster(args *xdsClusterArgs) *clusterv3.Cluster {
+	dnsLookupFamily := clusterv3.Cluster_V4_ONLY
```

Comment:
What if "*args.ipFamily == egv1a1.IPv6"? Do we want to support IPv6 single stack? @arkodg @zirain

Reply:
this field is tied to the cluster/endpoint config for control plane communication b/w envoy proxy and envoy gateway, its safe to make this `clusterv3.Cluster_ALL` and rely on the DNS service to provide the right info

Reply:
@juwon8891 can we update this to `clusterv3.Cluster_ALL` by default ?